### ISSUE TITLE
Add more details about privacy

### ISSUE THREAD

**Issue by mg101** [tomlandia/fuckr#2]
Hi there, thanks for the great app! I was wondering if you could share some details about how the app handles personal information? For example, where is my personal data stored, is it on my machine or on some grindr server somewhere? Can I remove it or would I need the grindr app to do that---I don't see a 'delete account button'. I tried removing the app and reinstalling for example and I can see it remembers everything when I sign in again. Sorry I'm not a tech-guy, my friend setup this github thingy :)

**Comment by tomlandia** [comment]
Hello :).

This app has no server, it stores everything on your computer. As for Grindr servers:
* They don't store messages after delivery (which is annoying if you want to use it on two devices!)
* But they do store pics. You can delete your account using the mobile app but I don't know how long they keep them after that :/.

**Comment by mg101** [comment]
thanks for replying! so I'm a bit confused about why I can see all my messages and pictures again after removing and reinstalling the app, which folder does it keep all that stuff in? I guess not somewhere in the directory tree that's removed when you trash the app. thanks again!

**Comment by tomlandia** [comment]
Oh sorry I missed this question. Like with most desktop apps, your data are stored in your 'app data' folder: https://github.com/nwjs/nw.js/wiki/app#datapath

**Comment by mg101** [comment]
Thanks, I don't see anything there though. The app is installed and I see everything when I'm logged in but when I look in ~/Library/Application Support I don't see anything that looks related. Here's a screenshot, I checked all the folders and don't see anything (I also enabled showing of hidden files using 'defaults write com.apple.finder AppleShowAllFiles YES' in a terminal window and relaunching finder)

![untitled](https://cloud.githubusercontent.com/assets/14146283/9703625/d1997076-5481-11e5-9036-9c6950fa587b.png)

**Comment by tomlandia** [comment]
Ah this is '/Library/Application Support/': the '~' is a shortcut for your home folder. From Finder, you can press ⇧⌘G and paste ~/Library/Application Support/

**Comment by mg101** [comment]
Oh I see, yea I found it now. I experimented a bit: I closed the app and moved the folder somewhere else then opened the app again (now I have to login), all it remembers is the 'about' details. I lose all my conversations and when I start new ones I also see that the pictures are gone too---you mentioned that they store pics but that's not the case from my testing, maybe it takes some time for them to get them or something? Anyway if I then close the app and move/replace the app folder with the original one everything is back like before. 

So in summary: to delete my account from my laptop (I don't mind much about whatever grindr stores) I can just delete the app folder in ~/Library/Application Support/ If I also want to remove the stuff that grindr stores I can delete my account on the mobile app but like you said no-one really knows what happens.

Anyhoo, this works for me, thanks again!
